06-data-subsetting.Rmd

```diff
@@ -210,6 +215,22 @@ x[-which(names(x) %in% c("a", "c"))]
 The `%in%` goes through each element of its left argument, in this case the
 names of `x`, and asks, "Does this element occur in the second argument?".
 
+> #### Tip: Non-unique names {.callout}
+>
+> You should be aware that it is possible for multiple elements in a
+> vector to have the same name. For example:
```

Comment:
Perhaps important to point out here that this is **not** the case for a `data.frame`?